README.md

```diff
@@ -1,10 +1,170 @@
+  clean and reusable by other projects.
+
+## rust-vmm crates
+
+Keep in mind that this status might not always be up to date. For the latest
+status please check individual repositories.
+
+### Coming Soon
+
+These are the empty repositories that have PRs waiting to be merged.
+
+- [linux-loader](https://github.com/rust-vmm/linux-loader): parser and loader
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
+  bindings to virtio kernel headers generated using
+  [bindgen](https://crates.io/crates/bindgen).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
+  implementation for virtio primitives such as virtqueues and descriptor chain.
+- [vmm-sys-utils](https://github.com/rust-vmm/vmm-sys-util/): a collection of
+  modules providing helpers and utilities for building VMMs and Hypervisors.
+- [vmm-vcpu](https://github.com/rust-vmm/vmm-vcpu/): a hypervisor-agnostic
+  abstraction for Virtual CPUs (vCPUs).
+
+### Existing crates
+
+These are crates for which we have most of the code out there, but
+there might be some missing requirements for publishing them on crates.io.
+For the requirement list refer to the
+[Publishing on crates.io - Requirements list](#publishing-on-crates.io---requirements-list)
+section.
+
+- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
+  Virtual Machine memory.
+
+#### Components Published on [crates.io](crates.io)
+
+These are production-ready components that can be pulled as dependencies
+directly from crates.io.
+
+- [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
+  to KVM generated using [bindgen](https://crates.io/crates/bindgen).
+- [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
+  KVM API.
+
+## How to contribute
+
+### Adding a new virtualization component
+
+If you have an idea about a component that could be a great fit for rust-vmm,
+please open a
+[review request issue](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=repository-review-request.md&title=Repository%2FCrate+Review+Request).
+The issue must provide the explanations and justifications behind the need for
+a new component, highlighting the purpose of this new repository, how will it
+be used by other VMMs or Hypervisors, and other information that you find
+relevant.
+
+Discussions about the proposal will happen on the same issue, and once an
+agreement is reached regarding the name (this part is hard!) and the design,
+an empty repository will be created in the rust-vmm GitHub organization.
+
+To add functionality you will need to send Pull Requests (PRs) against the
+newly created repository. While we do not expect the code to be perfect from
+the first try, we do want to keep a high standard in terms of quality for all
+existing repositories under rust-vmm.
+
+### Become a member of rust-vmm
```

Comment:
I suggest we stop doing this. Or at least we explain what it means to become a member of the org and what it brings in terms of permissions, etc. And we make it clear that non members can contribute as well.